index.d.ts

```diff
@@ -520,7 +521,7 @@ declare namespace Moleculer {
 		disconnected(): void;
 	}
 
-	class Context<P = unknown, M extends object = {}> {
```

Comment:
Please explain why you change it and it can break other TS users' projects.

Reply:
This was made `unknown` a while ago because with a default of `any` you, by default, get no type safety with moleculer context.